redaxo/src/core/fragments/core/form/searchfield.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$id = '';
+$class = 'input-group input-group-xs has-feedback form-clear-button';
+$clear = rex_i18n::msg('search_clear');
+$placeholder = ' placeholder = "' . rex_i18n::msg('search_placeholder') . '"';
+
+if ($this->id) {
+    $id = ' id = "' . $this->id .'"';
+}
+if ($this->placeholder) {
+    $placeholder = ' placeholder = "' . $this->placeholder . '"';
```

Comment:
```suggestion
    $placeholder = ' placeholder="' . $this->placeholder . '"';
```

Alternative
`$placeholder = ' placeholder="'.$this->getVar('placeholder', rex_i18n::msg('search_placeholder')).'"';`
Damit würdest du dir auch die Zeile 6 ersparen.